Changelog.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 **latest**
+- Add hook directory for user customizations to be mounted into the container
```

Comment:
`Add custom scripts directory `/data/toran-proxy/scripts` for user customizations to be executed on startup`